src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -244,7 +244,7 @@ export function getFeatureLayers(featuresConfig: FeaturesConfig) {
 			result += `
 				
 RUN cd /tmp/build-features/${feature.consecutiveId} \\
-&& export $(cat devcontainer-features.env | xargs) \\
+&& . ./devcontainer-features.env \\
```

Comment:
You could use `set -a` to export all assignments and then `set +a` to turn that off as in https://unix.stackexchange.com/a/79084. That would keep the env file simpler. Also seems to work with /bin/sh. @Chuxel used this in some of the scripts.

Reply:
Yeah, it's the simplest method I've found.